Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json

```diff
@@ -360,7 +360,7 @@
         {
           "RequestInputs": {
             "EndpointId": "$(ServiceNowConnection)",
-            "EndpointUrl": "$(endpoint.url)/api/now/table/change_request?x_mioms_azpipeline_stage_id=$(system.stageId)&x_mioms_azpipeline_build_id=$(build.buildId)&x_mioms_azpipeline_stage_attempt=$(checks.stageAttempt)&sysparm_fields=number,sys_id,correlation_id",
+            "EndpointUrl": "$(endpoint.url)/api/now/table/change_request?x_mioms_azpipeline_stage_id=$(checks.stageId)&x_mioms_azpipeline_build_id=$(build.buildId)&x_mioms_azpipeline_stage_attempt=$(checks.stageAttempt)&sysparm_fields=number,sys_id,correlation_id",
```

Comment:
Would be great if you can add in the description what the new stageId will look like.